examples/layersColorVisible.html

```diff
@@ -0,0 +1,63 @@
+<!-- this example outputs in console, the array of colors layers really displayed in viewer -->
```

Comment:
IMHO examples should demo an interesting visual feature.
I don't think this one is worth it.